cont3xt/overview.js

```diff
@@ -246,6 +346,20 @@ class Overview {
     for (const overview of overviews) {
       overview._editable = overview.creator === req.user.userId || req.user.hasRole(overview.editRoles);
       overview._viewable = overview.creator === req.user.userId || req.user.hasRole(overview.viewRoles);
+      for (const field of overview.fields) {
+        // take best guess at type in case it is missing
+        field.type ??= field.custom == null ? 'linked' : 'custom';
+
+        // custom is stored as a json string, so we parse it before sending back
+        if (field.type === 'custom') {
+          try {
+            field.custom = JSON.parse(field.custom);
+          } catch {
+            // we'd rather not crash the service if the custom field is somehow invalid
+            field.custom = '_cont3xt_INVALID_CUSTOM_FIELD';
```

Comment:
Do we want to kill the user's custom field json or send an error for them to correct their json?

Reply:
hmm, I'm not quite sure what the best choice is.
I'm pretty sure this should already be technically impossible since our verifier should be ensuring that our json is valid (and thus parseable). 
I just thought I would add this in case something changes or goes terribly wrong down the road.
We could just fully remove the field I guess. Alternatively an error for the user is also an option, but I think that might be complicating the response and it's handling even more for a result that theoretically shouldn't occur in the first place.
The reason I like this approach is that it still lets you know that something is wrong if you dig into it, without us needing to add some sort of type:error or consider the case where custom doesn't exist (because it really always should).
Maybe none of this is necessary at all! I just find JSON parsing on the server a little scary

Reply:
I agree, JSON parsing is always a little scary 😱 Since we previously ensure the JSON is valid (I missed that), then this should be fine! I like that you're catching this just in case (even though it is unlikely to get in this catch).

Thank you for helping me understand!